src/resolvers/mutation.js

```diff
@@ -13,10 +16,23 @@ export const mutationResolvers = {
     UpdateControlAction (object, params, ctx, resolveInfo) {
       const queryGenerator = new UpdateControlActionQuery(params, resolveInfo)
       return runQuery(queryGenerator.query, 'UpdateControlAction', 'ControlActionMutation')
-    }
+    },
+    CreateMediaObject: QueryAndPublishResolver.createResolver('MediaObject', 'MediaObjectCreateMutation'),
+    CreateVideoObject: QueryAndPublishResolver.createResolver('VideoObject', 'VideoObjectCreateMutation'),
+    CreateAudioObject: QueryAndPublishResolver.createResolver('AudioObject', 'AudioObjectCreateMutation')
   }
 }
 
+// add resolvers for all create mutations
+walkSync(`${__dirname}/../schema/type`, { directories: false, includeBasePath: true, globs: ['*.graphql'] })
+  .forEach(file => {
+    const { name } = parse(file)
```

Comment:
In the case where we have file with multiple type definitions in it (https://github.com/trompamusic/ce-api/issues/93#issuecomment-599540992), it looks like this won't do exactly what we expect.
What's the alternative here? We would have to actually load these files and parse all of the types out of them. Is there a graphql library that allows us to do this?

Reply:
Good point @alastair, I could use the same `graphql-tools` function to iterate all types.

https://github.com/trompamusic/ce-api/pull/98/files#diff-1b21ae31202733dc0c9c48a31aa51040R19-R33

Reply:
@alastair this has been fixed in the latest commit https://github.com/trompamusic/ce-api/pull/51/commits/e0fc5265df8ba7874f9c14d4f8dee815cbaab239

Reply:
great. I'm updating our python library to support these subscriptions today to test the rest of the functionality

Reply:
Cool, I will merge this feature so we can get things going on our end. I will create a separate PR for any feedback you might have.

Thanks!